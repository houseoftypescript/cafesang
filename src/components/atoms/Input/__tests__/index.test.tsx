import { render } from '@testing-library/react';
import { Input } from '../index';

describe('Input', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Input
        id="id"
        name="name"
        placeholder="Placeholder"
        value="value"
        onChange={() => {}}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
