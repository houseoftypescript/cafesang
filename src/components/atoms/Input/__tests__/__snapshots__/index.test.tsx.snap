// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input to match snapshot 1`] = `
<div>
  <input
    class="px-4 py-2 rounded w-full shadow-2xl"
    id="id"
    name="name"
    placeholder="Placeholder"
    value="value"
  />
</div>
`;
